src/phpDocumentor/Reflection/Php/Factory/File/FlySystemAdapter.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Reflection\Php\Factory\File;
```

Comment:
We want to move this to the `phpDocumentor\Infrastructure\*` namespace since this is not part of our domain model (the interface is; because that is specific in our domain but this concrete implementation is specific to Flysystem). I could understand if the distinction is not entirely clear, please indicate if that is the case then I will try to elaborate